### ISSUE TITLE
eigen3/Eigen/Dense not found

### ISSUE THREAD

**Issue by jankap** [pradeepunique1989/UR5Robot-MEX-Matlab#2]
Hi :)

Unfortunately, `run_me.m` does not compile, see error below

```
>> run_me
Building with 'MinGW64 Compiler (C++)'.
Error using mex
In file included from C:\Temp\pradeepr-roboticist-UR5Robot-MEX-Matlab-150a138\ur_kinematics\ur5_kinematics_mex.cpp:11:0:
C:\Temp\pradeepr-roboticist-UR5Robot-MEX-Matlab-150a138\ur_kinematics\ur5_kinematics_mex.h:10:30: fatal error: eigen3/Eigen/Dense: No such file
or directory
 #include <eigen3/Eigen/Dense>
                              ^
compilation terminated.


Error in compile_ur5kinematics (line 8)
mex -R2018a -DUR5_PARAMS -I./ -I./class_interface/class_interface/ ur5_kinematics_mex.cpp ur_kin.cpp

Error in run (line 91)
evalin('caller', strcat(script, ';'));

Error in run_me (line 10)
run compile_ur5kinematics.m
```

Do I have to download the eigen lib or should it be shipped with Matlab?

Thanks.

**Comment by pradeepr-roboticist** [comment]
Yes. You have to download the eigen library. If you are using Ubuntu, it is quite easy to install it.
You can use the command "sudo apt install libeigen3-dev".

**Comment by jankap** [comment]
Nice, thanks, it's working.

I had to download eigen from https://bitbucket.org/eigen/eigen/downloads/ and extract it into UR5Robot-MEX-Matlab (this repo's root) and renamed it to `eigen3`.

To get it built, I then needed to include the eigen header and define M_PI.

`mex -R2018a -DUR5_PARAMS -I./ -I./class_interface/class_interface/ -I../ -DM_PI=3.141592653589793 ur5_kinematics_mex.cpp ur_kin.cpp`

And there it is 
![image](https://user-images.githubusercontent.com/12893844/65284472-e5691a00-db39-11e9-8cf7-9af6ce9897e6.png)


I'm on Windows 10, Matlab 2019b, MinGW64 Compiler (from Matlab addons)

Thanks a lot! I'll have a look at the rendering.
